Client/Frontend/Home/HomePanelViewController.swift

```diff
@@ -17,7 +17,7 @@ private struct HomePanelViewControllerUX {
 }
 
 protocol HomePanelViewControllerDelegate: class {
-    func homePanelViewController(homePanelViewController: HomePanelViewController, didSelectURL url: NSURL, visitType: VisitType)
+    func homePanelViewController(homePanelViewController: HomePanelViewController, didSelectURL url: NSURL, visitType: VisitType, inNewTab: Bool)
```

Comment:
Unfortunately, Swift doesn't allow default parameters in protocol methods so I had to adapt all callers of this method.